commit 69e6b3e631fc8cd88cbe78b06d908f4310aa56e0
Author: q66 <daniel@octaforge.org>
Date:   Sat Dec 11 21:22:44 2021 +0100

    remove dependency on sentry
    
    This is unused and breaks ppc64 because it downlaods binaries.

diff --git a/package.json b/package.json
index 078c772..5b5c430 100644
--- a/package.json
+++ b/package.json
@@ -89,7 +89,6 @@
     "@babel/runtime": "^7.12.5",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.4.3",
     "@principalstudio/html-webpack-inject-preload": "^1.2.7",
-    "@sentry/webpack-plugin": "^1.18.1",
     "@types/flux": "^3.1.9",
     "@types/jest": "^27.0.2",
     "@types/modernizr": "^3.5.3",
diff --git a/webpack.config.js b/webpack.config.js
index 3aed215..7babcef 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -9,7 +9,6 @@ const TerserPlugin = require('terser-webpack-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const HtmlWebpackInjectPreload = require('@principalstudio/html-webpack-inject-preload');
 const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
-const SentryCliPlugin = require("@sentry/webpack-plugin");
 
 dotenv.config();
 let ogImageUrl = process.env.RIOT_OG_IMAGE_URL;
@@ -539,12 +538,6 @@ module.exports = (env, argv) => {
             }),
             useHMR && new ReactRefreshWebpackPlugin(fullPageErrors ? undefined : { overlay: { entry: false } }),
 
-            // upload to sentry if sentry env is present
-            process.env.SENTRY_DSN &&
-                new SentryCliPlugin({
-                    release: process.env.VERSION,
-                    include: "./webapp/bundles",
-                }),
             new webpack.EnvironmentPlugin(['VERSION']),
         ].filter(Boolean),
 
